v-next/hardhat-keystore/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import type { HardhatPlugin } from "@ignored/hardhat-vnext/types/plugins";
+
+import { task } from "@ignored/hardhat-vnext/config";
+import { ArgumentType } from "@ignored/hardhat-vnext/types/arguments";
+
+import "./internal/type-extensions.js";
+
+import { PLUGIN_ID } from "./internal/constants.js";
```

Comment:
What should the pattern be here for PLUGIN_ID? It is used across the plugin - are we happy with this pattern of `constants.js` and importing it here?